README.md

```diff
@@ -89,38 +87,33 @@ to run a simple example with Alluxio.
 
 ## Report a Bug
 
-To report bugs, suggest improvements, or create new feature requests, please open a [Github Issue](https://github.com/alluxio/alluxio/issues). Our previous [Alluxio JIRA system](https://alluxio.atlassian.net) has been deprecated since December 2018.
+To report bugs, suggest improvements, or create new feature requests, please open a [Github Issue](https://github.com/alluxio/alluxio/issues).
+If you are not sure whether you run into bugs or simply have general questions with respect to Alluxio, post your questions on [Alluxio Slack channel](www.alluxio.io/slack).
 
 ## Depend on Alluxio
 
 Alluxio project provides several different client artifacts for external projects to depend on Alluxio client:
 
-- Artifact `alluxio-core-client-fs` provides a client for
+- Artifact `alluxio-core-client-fs` provides a Java artifact for
   [Alluxio Java file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#alluxio-java-api))
   to access all Alluxio-specific functionalities.
-- Artifact `alluxio-core-client-hdfs` provides a client for
+- Artifact `alluxio-core-client-hdfs` provides a Java artifact for
   [HDFS-Compatible file system API](https://docs.alluxio.io/os/user/stable/en/api/FS-API.html#hadoop-compatible-java-client).
-- Artifact `alluxio-shaded-client`, available in Alluxio 2.0.1 or later, includes both above
-  clients and all their dependencies but in a shaded form to prevent conflicts. This client jar is
-  self-contained but the size is also much larger than the other two. This artifact is recommended
-  generally for a project to use Alluxio client.
+- (Recommended) Artifact `alluxio-shaded-client` includes both artifact. 
```

Comment:
nit: move up `alluxio-shaded-client` as the first one of client artifacts?

Reply:
done